cuenca/resources/api_keys.py

```diff
@@ -37,5 +38,20 @@ def deactivate(cls, api_key_id: str, minutes: int = 0) -> 'ApiKey':
         exact deactivated_at time.
         """
         url = cls._resource + f'/{api_key_id}'
-        resp = session.delete(url, dict(minutes=minutes))
+        resp = global_session.delete(url, dict(minutes=minutes))
         return cast('ApiKey', cls._from_dict(resp))
+
+    @classmethod
+    def update(
+        cls,
+        api_key_id: str,
+        metadata: Optional[dict] = None,
+        user_id: Optional[str] = None,
+    ) -> 'ApiKey':
+        """
+        If the current user has enough permissions, it associates an ApiKey to
+        the `user_id` or updates the correspoding metadata
+        """
+        req = ApiKeyUpdateRequest(metadata=metadata, user_id=user_id)
+        resp = cls._update(api_key_id, **req.dict(exclude_none=True))
```

Comment:
what about making `exclude_none=True` the default for all request validations the way we did for queries: https://github.com/cuenca-mx/cuenca-validations/blob/main/cuenca_validations/types/queries.py#L32

Reply:
it make sense, I will send a PR for that in `cuenca-validations`